steps/generate_dataset_from_htme.py

```diff
@@ -332,11 +334,24 @@ def get_plaintext_key_calling_dks(
     return key
 
 
+def retry_requests(retries=10, backoff=1):
+    retry_strategy = Retry(
+        total=retries,
+        backoff_factor=backoff,
+        status_forcelist=[429, 500, 502, 503, 504]
```

Comment:
having constants for code with meaning ful names will be more readable
INTERNAL_SERVER_ERROR = 500

Reply:
i think HTTP status codes are more readable than ful names

Reply:
like status 200 is more readable than OK i think